README.md

```diff
@@ -143,4 +143,47 @@ I would like to thank Microverse for the project snapshot from [Zeplin]( https:/
 
 This project is [MIT](./LICENSE) licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [
+    {
+      itemId: '1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      itemId: '2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      itemId: '3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, { payload: { itemId, title, author } }) => {
+      state.books.push({ itemId, title, author });
+    },
+    removeBook: (state, { payload: { itemId } }) => {
+      state.books = state.books.filter((book) => book.itemId !== itemId);
+    },
+  },
+});
+
+export const { addBook } = bookSlice.actions;
+
+export default bookSlice.reducer;
```

Comment:
[OPTIONAL] I suggest you remove this chunk from your README to make it more professional.

Reply:
Hello @laiifuu ,

Thank you, I have done that.